core/src/main/java/com/linecorp/armeria/client/HttpHeaderUtil.java

```diff
@@ -20,11 +20,14 @@
 
 import io.netty.util.AsciiString;
 
-final class HttpHeaderUtil {
+/**
+ * Provides utility functions for internal use related with HTTP headers.
+ */
+public final class HttpHeaderUtil {
```

Comment:
If made public, should move to the `internal` package.

Reply:
Moved to `internal` package.